src/components/CallKeyPad/index.tsx

```diff
@@ -62,12 +72,25 @@ const KeyPad: FC<{
     ))}
 
     <View style={[styles.keyPadNumber, styles.footerButtons]}>
-      <CallButtons
-        onPress={callVoice}
-        image={CustomImages.CallAcceptedLogo}
-        containerStyle={styles.callButtonContainer}
-        imageStyle={styles.callButtonImage}
-      />
+      <RnTouchableOpacity style={styles.wrapper} onPress={onHidePress}>
+        <RnText style={styles.hideWrapper}>{'Hide'}</RnText>
+      </RnTouchableOpacity>
+      {!duringCall && (
+        <CallButtons
+          onPress={callVoice}
+          image={CustomImages.CallAcceptedLogo}
+          containerStyle={styles.callButtonContainer}
+          imageStyle={styles.callButtonImage}
+        />
+      )}
+      {duringCall && !!hangup && (
```

Comment:
Remove double not in hangup. Use ternary operator in the statement with empty fragments instead.